cmd/modern/root.go

```diff
@@ -17,6 +17,7 @@ type Root struct {
 	configFilename string
 	loggingLevel   int
 	outputType     string
+	inputType      string
```

Comment:
>inputType      string [](http://example.com/codeflow?start=1&length=21)

why is this a root level parameter?

Reply:
This is a good question.  Where should we put all these back-compat switches.  They are needed either when the customer uses the `query` subcommand, or when the use uses no subcommand (which is defined in this Root type!).   

Should we add all these back-compat switches to the Query subcommand, and then have them accessable to the Root when no subcommand is provided.